src/GameState/index.tsx

```diff
@@ -356,7 +356,7 @@ function GameInfo(props: any) {
       </HStack>
       <Stack p={4} space={2}>
         <Text fontSize="lg" bold color="white">
-          Player turn: #{game?.currentPlayer}
+          Player turn: #{game?.currentPlayer | game?.currentPlayerId}
```

Comment:
I'm not sure it's a good idea to use the operator bitwise or `|`. You probably want to use the classic or `||`.